src/Contrib/Caching/Masa.Contrib.Caching.MultilevelCache.Tests/MultilevelCacheClientTest.cs

```diff
@@ -90,6 +90,31 @@ public async Task TestGetAndSubscribeAsync()
         await _multilevelCacheClient.RemoveAsync<string>(key);
     }
 
+    [TestMethod]
+    public async Task TestSubscribeChannelsAsync()
+    {
+        var subscribeChannelsField = _multilevelCacheClient.GetType().GetField("_subscribeChannels", BindingFlags.NonPublic | BindingFlags.Instance);
+
+        var key = "test200";
+        string? value = await _multilevelCacheClient.GetAsync<string>(key, newVal =>
+        {
+            value = newVal;
+        });
+        var subscribeChannelCount = ((List<string>)subscribeChannelsField!.GetValue(_multilevelCacheClient)!).Count;
+        Assert.AreEqual(subscribeChannelCount, 1);
+
+        await _multilevelCacheClient.RemoveAsync<string>(key);
+        subscribeChannelCount = ((List<string>)subscribeChannelsField!.GetValue(_multilevelCacheClient)!).Count;
+        Assert.AreEqual(subscribeChannelCount, 0);
+
+        await _multilevelCacheClient.GetAsync<string>(key, newVal =>
+        {
+            value = newVal;
+        });
+        subscribeChannelCount = ((List<string>)subscribeChannelsField!.GetValue(_multilevelCacheClient)!).Count;
+        Assert.AreEqual(subscribeChannelCount, 1);
```

Comment:
The expected value and the target value are wrongly written

```csharp
Assert.AreEqual(1, subscribeChannelCount);
```

Reply:
Missing unit tests for sync delete

Reply:
ok,already repaired